x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -549,8 +549,7 @@ export const VisualizationWrapper = ({
   if (localState.missingRefsErrors?.length) {
     // Check for access to both Management app && specific indexPattern section
     const { management: isManagementEnabled } = application.capabilities.navLinks;
-    const isIndexPatternManagementEnabled =
-      application.capabilities.management.kibana.indexPatterns;
+    const isIndexPatternManagementEnabled = application.capabilities.indexPatterns.save;
```

Comment:
Reading the issue this is intended to close it seems like we should stick with the `management.kibana.indexPatterns` capability here because it's used to determine whether or not a link to the data view management page is shown: https://github.com/elastic/kibana/pull/122503/files#diff-8ca1f3be4f6696d9c05773cfaa23d7489a1c460fffdf39c8ad2ba8aa09f6c194R561

<img width="619" alt="Screenshot 2022-01-11 at 11 00 05" src="https://user-images.githubusercontent.com/1508364/148921639-df3ddded-c597-4058-8e51-12bcd9f33d75.png">


As far as I understand having permissions to save a data view without showing the data view management page would still result in an error.